src/common/meta/src/kv_backend/txn.rs

```diff
@@ -146,6 +147,43 @@ impl Txn {
         self
     }
 
+    pub fn prepend_root(mut self, root: &[u8]) -> Self {
```

Comment:
Can we put this `prepend_root` function in `EtcdStore`'s Implementation for `TxnService`? Other kv backend Implementations don't need it.

Also in doing so, maybe the functions `chroot_key_value_with`, `key_strip_root` and `key_prepend_root` can be better placed in `EtcdStore`.

Reply:
The issue is `req: TxnRequest,` private. If following your suggestion, we should export this field to `pub(crate)`?

Reply:
That is, we need somehow modify the request (key).